develop_cmd.go

```diff
@@ -35,6 +35,9 @@ func (d *developCmd) contDeploy(e *env, deployer deployFunc, first, done chan st
 	ticker := e.Clock.Ticker(d.deployInterval)
 	defer ticker.Stop()
 
+	printFreq := int(20 * (time.Second * 1.0 / d.deployInterval))
```

Comment:
there should also be printFreq+=1
because say I want to print an error every 20s, arguable? perhaps 10s?
but if my deploy interval is 1 minute then printFreq=0 causing division by zero underneath.